src/ui/server/controllers/insurance/account/create/confirm-email/index.test.ts

```diff
@@ -66,6 +83,19 @@ describe('controllers/insurance/account/create/confirm-email', () => {
       });
     });
 
+    describe('when there is no accountIdToConfirm in the session and no ID query param', () => {
+      beforeEach(() => {
```

Comment:
very very small, if it's only one `it` statement, then this can be `beforeAll`?

Reply:
Great shout @IgnatG , beforeAll was causing some issues so i've just moved the before's entirely and have them inside the `it` statement.